### PR TITLE
Add OpenJ9 Java 8 packages for opening to unnamed module

### PR DESCRIPTION
**Pull request by pdbain-ibm**
Add legacy Java 8 OpenJ9 packages which are available by reflection from unnamed modules.
Fixes https://github.com/eclipse/openj9/issues/2062

Signed-off-by: Peter Bain <peter_bain@ca.ibm.com>

### CODE REVIEW

**Comment by pdbain-ibm** [comment]
FYI @pshipton

File: jdk/src/java.base/share/classes/jdk/internal/module/jdk8_packages.dat [hunk 1]
```diff
@@ -1,3 +1,9 @@
+/*
+ * ===========================================================================
+ * (c) Copyright IBM Corp. 2015, 2017 All Rights Reserved
```
Comment (pshipton) [inline jdk/src/java.base/share/classes/jdk/internal/module/jdk8_packages.dat]: Does this work? I think the IBM copyright should use the same comment character as the Oracle copyright. Also the dates should be 2018, 2018.

**Comment by pdbain-ibm** [comment]
Copyright dates fixed.
I followed the copyright format used in other files, e.g. jdk/src/java.base/share/classes/java/util/stream/Streams.java, though I'm happy to change it if @andrew-m-leonard wishes.

**Comment by pshipton** [comment]
While `/* */` is an accepted comment for .java code, I'm not so sure about this .dat file. I'd much prefer to use the known working comment character, otherwise its unclear what side affects the comment may have.

**Comment by pdbain-ibm** [comment]
Tested in manually built JDK:
```
82743a6aed11-root:162$ ./openj9-openjdk-jdk10/build/linux-x86_64-normal-server-release/jdk/bin/java -cp /team/pdbain/projects/attach/testlib/pdb.modularity.test.jar pdbtest.Attach
provider IBM class com.ibm.tools.attach.attacher.OpenJ9AttachProvider
public com.ibm.tools.attach.attacher.OpenJ9AttachProvider public name
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by pdbtest.Attach (file:/team/pdbain/projects/attach/testlib/pdb.modularity.test.jar) to method com.ibm.tools.attach.attacher.OpenJ9AttachProvider.name()
WARNING: Please consider reporting this to the maintainers of pdbtest.Attach
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
IBM
public abstract com.sun.tools.attach.spi.AttachProvider public abstract name
IBM
82743a6aed11-root:163$ head openj9-openjdk-jdk10/src/java.base/share/classes/jdk/internal/module/jdk8_packages.dat 
#
# ===========================================================================
# (c) Copyright IBM Corp. 2018, 2018 All Rights Reserved
# ===========================================================================
#

# Copyright (c) 2017, Oracle and/or its affiliates. All rights reserved.
# DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
#
# This code is free software; you can redistribute it and/or modify it
```
Note that OpenJDK overriding in personal builds is broken.

**Comment by pdbain-ibm** [comment]
Once this is approved I will copy the change to Java 10 & 11.

**Review by pshipton — APPROVED** [review]

**Comment by pshipton** [comment]
jenkins test sanity xlinux

**Comment by pdbain-ibm** [comment]
Sanity check failed due to JIT change: `/home/jenkins/workspace/PullRequest-Sanity-JDK9-linux_x86-64_cmprssptrs-OpenJDK9/build/linux-x86_64-normal-server-release/vm/compiler/../compiler/runtime/Runtime.cpp:178:77: error: no ‘void* TR_RuntimeHelperTable::getFunctionEntryPointOrConst(TR_RuntimeHelper)’ member function declared in class ‘TR_RuntimeHelperTable`

**Comment by pshipton** [comment]
jenkins test sanity xlinux

**Comment by pshipton** [comment]
Seems Eclipse ci is down. Will try again later.

**Comment by pshipton** [comment]
jenkins test sanity xlinux

**Pull request merged**